python/nano/src/bigdl/nano/tf/keras/__init__.py

```diff
@@ -19,3 +19,4 @@
 from .Sequential import Sequential
 from .Model import Model
 from .inference.optimizer import InferenceOptimizer
+from .customized_training_utils import tf_bf16_nano
```

Comment:
should we just call it `nano_bf16`

Reply:
> should we just call it `nano_bf16`

I‘m not sure whether to remain different decorators for cases of pytorch and tensorflow.